### DISCUSSION TITLE
Timeline component

### DISCUSSION THREAD

**Discussion by lauraptl** [vmware-clarity/ng-clarity#350] [Q&A]
Hello everybody!

I recently started working as a UX/UI designer and my company uses Angular for one of their software.
I would like to use the **timeline** component but I would like to replace existing icons by others (I selected some of the Clarity icons library).

Is it possible to **switch icons in this component**?

I'm sorry if my question is not relevant, i'm very new to this :)

Thank you for your help!

**Comment by ashleyryan** [comment]
If your devs use the Clarity UI version of the timeline (the HTML/CSS only implementation) then you can use a custom icon - you can see it in practice down towards the bottom of this page: https://clarity.design/documentation/timeline#examples

However, if they're already using the Angular component, it's not possible. And switching to the HTML/CSS version requires more work on their side to get the accessibility of the component to be correct.
